daemon/container.go

```diff
@@ -329,6 +329,11 @@ func (daemon *Daemon) verifyContainerSettings(platform string, hostConfig *conta
 		return nil, errors.Errorf("invalid restart policy '%s'", p.Name)
 	}
 
+	// validate isolation mode
+	if !hostConfig.Isolation.IsValid() {
```

Comment:
When running LCOW; is this assumption (validation based on _runtime_ platform) still valid? I.e., if I start a Linux container from a Windows daemon? (perhaps it's all good, but double-checking); other parts of the code in this function (`verifyContainerSettings()`) use, e.g.:

```go
if runtime.GOOS == platform {
```

Reply:
Isolation support for LCOW is checked there: https://github.com/moby/moby/blob/master/daemon/create_windows.go#L24